cmd/influxd/run.go

```diff
@@ -249,6 +257,24 @@ func initializeServer(s *influxdb.Server, b *messaging.Broker) {
 	}
 }
 
+func openDefaultServer(s *influxdb.Server, b *messaging.Broker) {
+	// TODO: Create replica using the messaging client.
+
+	// Create replica on broker.
+	if err := b.CreateReplica(1); err != nil {
```

Comment:
I'm not 100% sure.  I had a lot of questions on what some of this code did so I made some assumptions...  Hopefully Ben can shed some light on this.